addon/mixins/focusing.js

```diff
@@ -1,21 +1,28 @@
-import Ember from "ember";
-import getOwner from 'ember-getowner-polyfill';
+import Ember from 'ember';
 
-const { get } = Ember;
+/* globals getOwner */
+// One of these two will be uncommented at build time:
```

Comment:
@rwjblue exactly right. happy to PR `ember-getowner-polyfill` to address that. meanwhile, would you 👍 if I move it into a util module to land this PR?